sdk/identity/azure-identity/src/main/java/com/azure/identity/AuthorizationCodeCredentialBuilder.java

```diff
@@ -47,6 +50,40 @@ public AuthorizationCodeCredentialBuilder redirectUrl(String redirectUrl) {
         return this;
     }
 
+    /**
+     * Sets the client secret for the authentication.
+     * @param clientSecret the secret value of the AAD application.
+     * @return the AuthorizationCodeCredentialBuilder itself
+     */
+    public AuthorizationCodeCredentialBuilder clientSecret(String clientSecret) {
+        this.clientSecret = clientSecret;
+        return this;
+    }
+
+    /**
+     * Sets the client certificate for authenticating to AAD.
+     *
+     * @param certificatePath the PEM file containing the certificate
+     * @return the AuthorizationCodeCredentialBuilder itself
+     */
+    public AuthorizationCodeCredentialBuilder pemCertificate(String certificatePath) {
```

Comment:
Public API is being added.
Should update changelog to keep track of public API changes in this PR (InteractiveBrowserCredentialBuilder and AuthorizationCodeCredentialBuilder)

Reply:
Adding change log now